01_scripts/03_annotate_genes.py

```diff
@@ -59,6 +60,10 @@ def get_info(line):
                         l = line.replace("DR   Pfam;", "").strip()
                         l = l.split(";")[0]
                         self.pfam += l + ";"
+                    elif line.startswith("DR   KEGG;"):
+                    l = line.replace("DR   KEGG;", "").strip()
+                    l = l.split(";")[0]
+                    self.pfam += l + ";"
```

Comment:
This chunk needs a tabulation to be inside the elif condition. Also, I guess it's self.KEGG in the last line.

Unfortunately, the input file does not have KEGG information, so it's not possible to extract it, rendering this commit useless. 

It's a good idea, but will need a work around.